### ISSUE TITLE
consider time of privacy breach for blockchain-based attacks

### ISSUE THREAD

**Issue by kristovatlas** [OpenBitcoinPrivacyProject/wallet-ratings#127]
It may be helpful to break up attacks and/or countermeasures based on when privacy breaches actually take place.

For example, a change output might be attributed to you as a sender based on the transaction the output belongs to, or based on future transactions due to behavior of you and/or your counter-party. 

There are related countermeasures for both of these. Since users tend to stick with the same wallet and wallet client behavior tends not to change drastically over time, the countermeasures will often apply to both times of potential privacy breach, but this is not necessarily always the case.

**Comment by dcousens** [comment]
Is timing analysis an attack? 
It would definitely be a high entropy behavioural exposure...

For example,  say I gamble at night,  and buy coffee using bitcoin in the morning.  Near consistently.

If this is a pattern on even a daily basis,  it'd be very easy to isolate my addresses over time based on those two correlations.

**Comment by kristovatlas** [comment]
Yes, definitely. I also expect that it would be a high entropy approach, though. 

For reference, oxt.me provides temporal analysis on a per-address basis, but the same could easily be done for clusters of addresses: https://oxt.me/address/tiid/133522641

**Comment by kristovatlas** [comment]
Proposed attack (wording needs to be refined): Identify ownership of change output based on temporal patterns of output spends and how they coincide with temporal patterns of clustered addresses.

**Comment by LaurentMT** [comment]
> > For reference, oxt.me provides temporal analysis on a per-address basis, but the same could easily be done for clusters of addresses: https://oxt.me/address/tiid/133522641

Actually, OXT provides the same analysis on a per-entity basis too. :)

A few examples:
- BTCC (China): https://oxt.me/entity/BTCCHINA
- Multibit (UK) : https://oxt.me/entity/multibit
- Early adopter living in Texas : https://oxt.me/entity/tiid/439610376

A post on reddit explaining how to analyze these charts: https://www.reddit.com/r/Bitcoin/comments/42xcyp/privacy_do_bitcoiners_dream_of_electronic/
